GameDefinition/Content/ItemKind.hs

```diff
@@ -1739,3 +1739,24 @@ currency = currencyTemplate
                ++ iaspects currencyTemplate
   , ieffects = [RefillCalm (-1)]
   }
+  
+-- * Clothing
+
+smokingJacket = ItemKind
+  { isymbol  = _symbolClothes
+  , iname    = "smoking jacket"
+  , ifreq    = [("common item", 100), ("misc clothing", 1)]
```

Comment:
```suggestion
  , ifreq    = [("common item", 100), ("misc clothing", 1), ("chic gear", 100)]
```
How about we mark it so and then prefer generating such items in the Smoking room caves (and Casinos in Allure)? We might also mark some jewellery, etc., but let's be picky. :)

Reply:
All right. Now you can just take away half of the probability from the other groups on the smoking rooms levels and add the chic garments with the sum of what you took (that's rather a lot, but there's only one item in the chic group, so to have any specimen of the group on a level, we have bump it up):

https://github.com/LambdaHack/LambdaHack/blob/master/GameDefinition/Content/CaveKind.hs#L104